crates/bevy_pbr/src/alpha.rs

```diff
@@ -2,16 +2,19 @@ use bevy_ecs::{component::Component, reflect::ReflectComponent};
 use bevy_reflect::std_traits::ReflectDefault;
 use bevy_reflect::Reflect;
 
-// FIXME: This should probably be part of bevy_render2!
-/// Alpha mode
+/// Alpha mode.
```

Comment:
This isn't really a doc comment...
Maybe something like "configure how alpha blending is done"